src/flex/FlexView.js

```diff
@@ -76,6 +73,17 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    const { auto } = this.props;
+    if (!t.Nil.is(auto)) {
+      warn('"auto" is DEPRECATED. Use combination of "grow", "shrink" and "basis" instead');
```

Comment:
what I'm saying is it's not common to have warnings for features of old versions that are not there anymore (would be impossible to maintain such logs evolving a library..)

IMHO this should be explicit enough if we bump the semver and list the breaking changes somewhere, but also ccing @gabro here